supervisor_types.go

```diff
@@ -302,6 +302,19 @@ type HttpInputSourceConfig struct {
 	AllowedProtocols []string `json:" allowedProtocols,omitempty"`
 }
 
+// ConnectorConfig is connection configuration for Database
+type ConnectorConfig struct {
+	ConnectURI string `json:"connectURI"`
+	User       string `json:"user,omitempty"`
```

Comment:
not sure about this `omitempty` here - we need that field to progress

Reply:
I assumed username/password could be encoded in the connectURI. I will fix it though, since passing credentials in connect uri is less secure.